### PR TITLE
Most basic blog layout & setup.

### PR DESCRIPTION
**Pull request by n1k0**
I didn't spend too much time crafting th layout though that does the job.

I'd really appreciate either @DirtyF or @ticabri to try the branch and integrate their blog post to see how it goes. After cloning the repo:

```
$ make install
```

Update the `content/2016.12.hello-world.md`  with the markdown contents of the blog post. Then:

```
$ make regenerate
```

And head to [http://localhost:8000/news.html](http://localhost:8000/news.html). Every modification to the blog contents will be refreshed if you reload the page.

I'd also like a quick review from @almet.

### CODE REVIEW

File: content/pages/about.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <!-- Header -->
       <div id="header-wrapper">
         <div id="header" class="container">
-          <h1 id="logo"><img src="images/kinto-logo.svg" alt="Kinto"/></h1>
+          <h1 id="logo"><a href="./"><img src="images/kinto-logo.svg" alt="Kinto"/></a></h1>
```
Comment (almet) [inline content/pages/about.html]: If you want to point to the root, use `SITE_URL` instead.

File: pelicanconf.py [hunk 2]
```diff
@@ -18,12 +18,8 @@
 DEFAULT_PAGINATION = False
 
 THEME = "theme"
-INDEX_SAVE_AS = 'news.html'
+INDEX_SAVE_AS = 'index.html'
```
Comment (almet) [inline pelicanconf.py]: if you do that you'll replace the landing page by something else (the list of articles here).

File: theme/templates/base.html [hunk 3]
```diff
@@ -40,7 +46,7 @@
               <a class="icon fa-slack" href="https://slack.kinto-storage.org/" title="Slack channel">Slack</a>
             </li>
             <li>
-              <a class="icon fa-rss" href="http://www.kinto-storage.org/archives.html" title="Latest articles about Kinto">Blog</a>
+              <a class="icon fa-rss" href="./archives.html" title="Latest articles about Kinto">Blog</a>
```
Comment (almet) [inline theme/templates/base.html]: Use `SITE_URL` here as well.

Comment (almet) [inline theme/templates/base.html]: You might also point to `news.html` once you've restored this behaviour (not sure why you did this change).

File: content/pages/about.html (hunk 1 above)
Comment (n1k0) [inline content/pages/about.html]: I think I tried but that doesn't get replaced with the actual ur, it stays that way... not very convenient

File: pelicanconf.py (hunk 2 above)
Comment (n1k0) [inline pelicanconf.py]: ah! ok makes sense

File: theme/templates/base.html (hunk 3 above)
Comment (n1k0) [inline theme/templates/base.html]: I didn't get the difference between news.html and archives.html tbh :)

Comment (almet) [inline theme/templates/base.html]: they serve different purposes: in the archive you have all the articles whereas on the news they are paginated (but this of course depends on the theme)

Comment (almet) [inline theme/templates/base.html]: Here, `SITE_URL` will work because this is used by Jinja2.

**Comment by n1k0** [comment]
I'm gonna land this. People interested in contributing articles should open new PRs :)

**Pull request merged**